test/integration/targets/win_scheduled_task/tasks/triggers.yml

```diff
@@ -288,6 +288,82 @@
     that:
     - trigger_monthlydow_again is not changed
 
+- name: create trigger repetition (check mode)
+  win_scheduled_task:
+    name: '{{test_scheduled_task_name}}'
+    state: present
+    actions:
+    - path: cmd.exe
+    triggers:
+    - type: registration
+      repetition:
+      - interval: PT1M
+        duration: PT5M
+        stop_at_duration_end: yes
+  register: trigger_repetition_check
+  check_mode: yes
+
+- name: get result of create trigger repetition (check mode)
+  win_scheduled_task_stat:
+    path: \
+    name: '{{test_scheduled_task_name}}'
+  register: trigger_repetition_result_check
+
+- name: assert results of create trigger repetition (check mode)
+  assert:
+    that:
+    - trigger_repetition_check is changed
+    - trigger_repetition_result_check.task_exists == False
+
+- name: create trigger repetition
+  win_scheduled_task:
+    name: '{{test_scheduled_task_name}}'
+    state: present
+    actions:
+    - path: cmd.exe
+    triggers:
+    - type: registration
+      repetition:
+      - interval: PT1M
+        duration: PT5M
+        stop_at_duration_end: yes
+  register: trigger_repetition
+
+- name: get result of create trigger repetition
+  win_scheduled_task_stat:
+    path: \
+    name: '{{test_scheduled_task_name}}'
+  register: trigger_repetition_result
+
+- name: assert results of create trigger repetition
+  assert:
+    that:
+    - trigger_repetition is changed
+    - trigger_repetition_result.task_exists == True
+    - trigger_repetition_result.triggers|count == 1
+    - trigger_repetition_result.triggers[0].type == "TASK_TRIGGER_REGISTRATION"
+    - trigger_repetition_result.triggers[0].enabled == True
+    - trigger_repetition_result.triggers[0].repetition.repetition == True
```

Comment:
I've just fixed the win_scheduled_task_stat to actually return the correct values, the repetition key will look like the following now

```
repetition {
    "duration": "PT1M",
    "interval: "PT5M",
    "stop_at_duration_end": true
}
```

If the duration or interval are not set then the value would be `null`